parsers/promotedTitle.js

```diff
@@ -7,19 +7,12 @@ var parse = require('./lib/parse');
 var entities = require('entities');
 
 var postcount = 0;
+var error = 0;
 
-function tryPage(elem) {
-    try {
-        return elem.attr().onmouseover.replace(/.*\"http/, 'http').replace(/\".*/, '').replace(/\\/g, '');
-    } catch(error) {
-        return null;
-    };
-}
 function getPromotedTitle(snippet) {
 
 	var title;
 	var title_type;
-	var error = 0;
```

Comment:
Moved this in the global scope, so, as `postcount`, can be used to print the number of unsuccessful title extraction:

![image](https://cloud.githubusercontent.com/assets/89555/21207198/e5d5ee08-c24d-11e6-9a13-0f9fd30e75b8.png)